multi/imxrt-multi/uart.c

```diff
@@ -772,7 +772,9 @@ static void uart_initPins(void)
 
 void uart_klogCblk(const char *data, size_t size)
 {
-	libtty_write(&uart_common.uarts[uartPos[UART_CONSOLE - id_uart1]].tty_common, data, size, 0);
+#ifdef UART_CONSOLE
+	libtty_write(&uart_common.uarts[uartPos[UART_CONSOLE - 1]].tty_common, data, size, 0);
```

Comment:
Why change  `id_uart1` to `1`?

Reply:
`UART_CONSOLE` is the number of the UART being used for console (starting at 1), not an ID from `imxrt-multi.h`. It just happens that `id_uart1 == 1`, but if anyone moved these IDs around this code would stop working.

Reply:
Valid point